src/Features/CSharp/Portable/CodeLens/CSharpCodeLensDisplayInfoService.cs

```diff
@@ -165,6 +165,34 @@ public string GetDisplayName(SemanticModel semanticModel, SyntaxNode node)
             }
         }
 
+        public string ConstructFullName(SemanticModel semanticModel, SyntaxNode methodNode)
+        {
+            var name = semanticModel.GetDeclaredSymbol(methodNode).Name;
+            var parent = methodNode.Parent;
+            while (parent != null)
+            {
+                if (parent.IsKind(SyntaxKind.ClassDeclaration))
+                {
+                    name = semanticModel.GetDeclaredSymbol(parent).Name + (!string.IsNullOrEmpty(name) ? "+" + name : "");
```

Comment:
This looks like it's just building the metadata name.  SymbolDisplay should be able to handle this IIRC.